src/commands/restriction/account.ts

```diff
@@ -71,24 +71,24 @@ export default class extends ProfileCommand {
     @metadata
     execute(options: CommandOptions) {
         this.spinner.start();
-        const profile = this.getProfile(options);
-
-        options.address =  OptionsResolver(options,
-            'address',
-            () => this.getProfile(options).account.address.plain(),
-            'Introduce an address: ');
+        const profile = this.getProfile();
+        options.address = Address.createFromRawAddress(
```

Comment:
This seems wrong, review line 80: const address = Address.createFromRawAddress(options.address);